microsoft-edge/hosting/webview2/Tooling/WebDriver.md

```diff
@@ -0,0 +1,132 @@
+---
+description: Automate and test the WebView2 Control using Microsoft Edge Driver
+title: Automating and Testing WebView2 with Microsoft Edge Driver
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/17/2020
+ms.topic: reference
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, edge, ICoreWebView2, ICoreWebView2Host, Selenium, Microsoft Edge Driver
+---
+
+# Automating and Testing WebView2 with Microsoft Edge Driver
+Because WebView2 utilizes the Chromium web platform, WebView2 developers can take advantage of standard web tooling for debugging and automation. One such tool is the W3C [WebDriver](https://www.w3.org/TR/webdriver2/) API, which can be used to create automated tests that simulate user interactions.
```

Comment:
To be super precise, I think this could actually be:
`One such tool is Selenium, which implements the W3C [WebDriver](https://www.w3.org/TR/webdriver2/) API, which can be used...`